doc/classes/Particles2D.xml

```diff
@@ -40,7 +40,7 @@
 			How rapidly particles in an emission cycle are emitted. If greater than [code]0[/code], there will be a gap in emissions before the next cycle begins. Default value: [code]0[/code].
 		</member>
 		<member name="fixed_fps" type="int" setter="set_fixed_fps" getter="get_fixed_fps">
-			The particle system's frame rate is fixed to a value. For instance, changing the value to 2 will make the particles render at 2 frames per second. Note this does not slow down the particle system itself.
+			The particle system's frame rate is fixed to a value. For instance, changing the value to 2 will make the particles render at 2 frames per second. Note this does not slow down the simulation of the particle system itself.
```

Comment:
This is actually reversed. Setting ``fixed_fps`` sets the speed at which the simulation occurs but does not impact the rate at which they are rendered. So if the game is running at 60 fps and ``fixed_fps`` is set to ``30`` the particles will update 30 times per second, but still be rendered 60 times per second.

Setting ``fixed_fps`` is a good way to save computation time for things that move slowly/infrequently.

This will have to be fixed for all the \*Particles\* classes

Reply:
To me Chris's docstring is correct: fixed fps does control the framerate of the particles without changing the system's simulation speed, that is to stay the node's time scale.

In other words, it changes the frequency at which the particle system updates.

Reply:
Not really my wording - this was the description already in the Particles2D doc, I just transferred it to the matching property on this node.

Reply:
@NathanLovato I still think stating that ``For instance, changing the value to 2 will make the particles render at 2 frames per second. Note this does not slow down the simulation of the particle system itself.`` Is misleading. To me, rendering is the actual drawing to screen, rendering at 2 frames per second means the object is drawn only 2 times per second. While simulation, to me, means the speed at which the particle data is updated.

@cbscribe I know it wasn't your wording. I was just pointing it out as something that could be fixed while we are on the topic of \*Particles\*. :)